### ISSUE TITLE
Appium switch to web view failed for Hybrid App

### ISSUE THREAD

**Issue by cindysongzi** [appium/appium#12713]
## The problem
driver.switch_to.context('WEBVIEW_com')

Android 7.0 (OPPO R11):
raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Failed to start Chromedriver session: An unknown server-side error occurred while processing the command. Original error: unknown error: Failed to get PID for the following process: com.test
  (process name must be specified if not equal to package name)
  (Driver info: chromedriver=2.46.628402 (536cd7adbad73a3783fdc2cab92ab2ba7ec361e1),platform=Windows NT 10.0.17763 x86_64)

Android 9.0 (HUAWEI Mate Pro 20):
raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Unable to automate Chrome version because it is too old for this version of Chromedriver.
Chrome version on the device: Chrome/70.0.3538.110
Visit 'https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/web/chromedriver.md' to troubleshoot the problem.
Failed to start Chromedriver session: A new session could not be created. Details: session not created: Chrome version must be between 71 and 75
  (Driver info: chromedriver=2.46.628402 (536cd7adbad73a3783fdc2cab92ab2ba7ec361e1),platform=Windows NT 10.0.17763 x86_64)

I have installed Chrome V74 in my HUAWEI Mate Pro 20 phone, actually. And it still fails even after I ungraded my chromedriver version manully (Path: \Appium\resources\app\node_modules\appium\node_modules\appium-chromedriver\chromedriver\win).

My setup script:
    def setUp(self):
        desired_caps = {}
        desired_caps['platformName'] = 'Android'
        desired_caps['deviceName'] = self.get_device_name()
        desired_caps['app'] = os.getcwd()+'\\app\\test.apk'
        desired_caps['automationName'] = 'UiAutomator1'
        desired_caps['appPackage'] = 'com.test'
        desired_caps['appActivity'] = 'com.test'
        desired_caps['noReset'] = 'true'
        desired_caps['chromeOptions.androidPackage'] = "com.android.chrome"
        self.driver = webdriver.Remote('http://localhost:4723/wd/hub', desired_caps)

## Environment

* Appium 1.13
* Desktop OS/version used to run Appium: Windows NT 10.0.17763 x86_64
* Node.js version (unless using Appium.app|exe): 10.15.3
* Npm or Yarn package manager: 6.4.1
* Mobile platform/version under test: Android 7.0 (OPPO R11) / Android 9.0 (HUAWEI Mate Pro 20)
* Real device or emulator/simulator: Real device
* Appium CLI or Appium.app|exe: Appium CLI
* JDK 12.0.1
* Python 3.6

**Comment by mykola-mokhnach** [comment]
- Google SDK is known to NOT support newer JDK
- Is the web view debuggable via Chrome Remote debugger?
- Have you read the tutorial above? Does the driver version match to web view version?

Also, the full server logs is required.

**Comment by cindysongzi** [comment]
Thanks @mykola-mokhnach .

Here is the appium server log:
$ appium -a 127.0.0.1 -p 4723
[Appium] Welcome to Appium v1.13.0
[Appium] Non-default server args:
[Appium]   address: 127.0.0.1
[Appium] Appium REST http interface listener started on 127.0.0.1:4723
[HTTP] --> POST /wd/hub/session
[HTTP] {"capabilities":{"firstMatch":[{"platformName":"Android","appium:deviceName":"f7cfcaf9","appium:app":"C:\\1_Automation\\FM3\\app\\evckids-live-debug-1.2.0.3300.apk","appium:automationName":"UiAutomator1","appium:appPackage":"com.ef.evc2015.kids","appium:appActivity":"com.ef.evc2015.SplashActivity","appium:noReset":"true","appium:chromeOptions.androidPackage":"com.android.chrome"}]},"desiredCapabilities":{"platformName":"Android","deviceName":"f7cfcaf9","app":"C:\\1_Automation\\FM3\\app\\evckids-live-debug-1.2.0.3300.apk","automationName":"UiAutomator1","appPackage":"com.ef.evc2015.kids","appActivity":"com.ef.evc2015.SplashActivity","noReset":"true","chromeOptions.androidPackage":"com.android.chrome"}}
[debug] [W3C] Calling AppiumDriver.createSession() with args: [{"platformName":"Android","deviceName":"f7cfcaf9","app":"C:\\1_Automation\\FM3\\app\\evckids-live-debug-1.2.0.3300.apk","automationName":"UiAutomator1","appPackage":"com.ef.evc2015.kids","appActivity":"com.ef.evc2015.SplashActivity","noReset":"true","chromeOptions.androidPackage":"com.android.chrome"},null,{"firstMatch":[{"platformName":"Android","appium:deviceName":"f7cfcaf9","appium:app":"C:\\1_Automation\\FM3\\app\\evckids-live-debug-1.2.0.3300.apk","appium:automationName":"UiAutomator1","appium:appPackage":"com.ef.evc2015.kids","appium:appActivity":"com.ef.evc2015.SplashActivity","appium:noReset":"true","appium:chromeOptions.androidPackage":"com.android.chrome"}]}]
[debug] [BaseDriver] Event 'newSessionRequested' logged at 1559563815760 (20:10:15 GMT+0800 (GMT+08:00))
[Appium] Appium v1.13.0 creating new AndroidDriver (v4.15.1) session
[Appium] Capabilities:
[Appium]   platformName: Android
[Appium]   deviceName: f7cfcaf9
[Appium]   app: C:\1_Automation\FM3\app\evckids-live-debug-1.2.0.3300.apk
[Appium]   automationName: UiAutomator1
[Appium]   appPackage: com.ef.evc2015.kids
[Appium]   appActivity: com.ef.evc2015.SplashActivity
[Appium]   noReset: true
[Appium]   chromeOptions.androidPackage: com.android.chrome
[debug] [BaseDriver] W3C capabilities {"alwaysMatch":{"platformNa... and MJSONWP desired capabilities {"platformName":"Android","... were provided
[debug] [BaseDriver] Creating session with W3C capabilities: {"alwaysMatch":{"platformNa...
[BaseDriver] Capability 'noReset' changed from string to boolean. This may cause unexpected behavior
[BaseDriver] The following capabilities were provided, but are not recognized by appium: chromeOptions.androidPackage.
[BaseDriver] Session created with session id: 02d27be2-f024-4f9e-9c3e-2c919b262c6c
[ADB] Found 1 'build-tools' folders under 'C:\AppData\Local\Android\Sdk' (newest first):
[ADB]     C:/Users/AppData/Local/Android/Sdk/build-tools/28.0.3
[ADB] Using 'adb.exe' from 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe'
[AndroidDriver] Retrieving device list
[debug] [ADB] Trying to find a connected android device
[debug] [ADB] Getting connected devices...
[debug] [ADB] 1 device(s) connected
[AndroidDriver] Using device: f7cfcaf9
[ADB] Using 'adb.exe' from 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe'
[debug] [ADB] Setting device id to f7cfcaf9
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 shell getprop ro.build.version.sdk'
[debug] [ADB] Current device property 'ro.build.version.sdk': 25
[debug] [ADB] Device API level: 25
[AndroidDriver] Consider setting 'automationName' capability to 'uiautomator2' on Android >= 6, since UIAutomator framework is not maintained anymore by the OS vendor.
[BaseDriver] Using local app 'C:\1_Automation\FM3\app\evckids-live-debug-1.2.0.3300.apk'
[debug] [AndroidDriver] Checking whether app is actually present
[AndroidDriver] Starting Android session
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 wait-for-device'
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 shell echo ping'
[debug] [AndroidDriver] Pushing settings apk to device...
[debug] [ADB] Getting install status for io.appium.settings
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 shell dumpsys package io.appium.settings'
[debug] [ADB] 'io.appium.settings' is installed
[debug] [ADB] Getting package info for 'io.appium.settings'
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 shell dumpsys package io.appium.settings'
[ADB] Using 'aapt.exe' from 'C:\AppData\Local\Android\Sdk\platform-tools\aapt.exe'
[debug] [ADB] The version name of the installed 'io.appium.settings' is greater or equal to the application version name ('2.14.0' >= '2.14.0')
[debug] [ADB] There is no need to install/upgrade 'C:\AppData\Roaming\npm\node_modules\appium\node_modules\io.appium.settings\apks\settings_apk-debug.apk'
[debug] [ADB] Getting IDs of all 'io.appium.settings' processes
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 shell 'pgrep --help; echo $?''
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 shell pgrep \^appium\\.settings\$'
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 shell am start -W -n io.appium.settings/.Settings -a android.intent.action.MAIN -c android.intent.category.LAUNCHER -f 0x10200000'
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 shell appops set io.appium.settings android\:mock_location allow'
[debug] [Logcat] Starting logcat capture
[ADB] Getting device platform version
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 shell getprop ro.build.version.release'
[debug] [ADB] Current device property 'ro.build.version.release': 7.1.1
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 shell wm size'
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 shell getprop ro.product.model'
[debug] [ADB] Current device property 'ro.product.model': OPPO R11
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 shell getprop ro.product.manufacturer'
[debug] [ADB] Current device property 'ro.product.manufacturer': OPPO
[debug] [ADB] Getting install status for com.ef.evc2015.kids
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 shell dumpsys package com.ef.evc2015.kids'
[debug] [ADB] 'com.ef.evc2015.kids' is installed
[debug] [ADB] Getting package info for 'com.ef.evc2015.kids'
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 shell dumpsys package com.ef.evc2015.kids'
[debug] [ADB] The version name of the installed 'com.ef.evc2015.kids' is greater or equal to the application version name ('1.2.0' >= '1.2.0')
[debug] [ADB] There is no need to install/upgrade 'C:\1_Automation\FM3\app\evckids-live-debug-1.2.0.3300.apk'
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 shell rm -rf /data/local/tmp/strings.json'
[debug] [AndroidDriver] Extracting strings from apk
[debug] [AndroidDriver] C:\1_Automation\FM3\app\evckids-live-debug-1.2.0.3300.apk
[debug] [AndroidDriver] null
[debug] [AndroidDriver] C:\Users\CINDY~1.SON\AppData\Local\Temp\com.ef.evc2015.kids
[debug] [ADB] Extracting strings from for language: default
[ADB] Successfully extracted 154 strings from 'C:\1_Automation\FM3\app\evckids-live-debug-1.2.0.3300.apk' resources for '(default)' configuration
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 shell mkdir -p /data/local'
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 push C\:\\Users\\CINDY~1.SON\\AppData\\Local\\Temp\\com.ef.evc2015.kids\\strings.json /data/local/tmp'
[debug] [AndroidBootstrap] Watching for bootstrap disconnect
[debug] [ADB] Forwarding system: 4724 to device: 4724
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 forward tcp\:4724 tcp\:4724'
[debug] [UiAutomator] Starting UiAutomator
[debug] [UiAutomator] Moving to state 'starting'
[debug] [UiAutomator] Parsing uiautomator jar
[debug] [UiAutomator] Found jar name: 'AppiumBootstrap.jar'
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 shell mkdir -p /data/local'
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 push C\:\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\appium-android-driver\\bootstrap\\bin\\AppiumBootstrap.jar /data/local/tmp/'
[debug] [ADB] Attempting to kill all uiautomator processes
[debug] [ADB] Getting IDs of all 'uiautomator' processes
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 shell pgrep \^uiautomator\$'
[ADB] No 'uiautomator' process has been found
[debug] [UiAutomator] Starting UIAutomator
[debug] [ADB] Creating ADB subprocess with args: ["-P",5037,"-s","f7cfcaf9","shell","uiautomator","runtest","AppiumBootstrap.jar","-c","io.appium.android.bootstrap.Bootstrap","-e","pkg","com.ef.evc2015.kids","-e","disableAndroidWatchers",false,"-e","acceptSslCerts",false]
[debug] [UiAutomator] Moving to state 'online'
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Loading json...
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] json loading complete.
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Registered crash watchers.
[AndroidBootstrap] Android bootstrap socket is now connected
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 shell dumpsys window'
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Client connected
[AndroidDriver] Screen already unlocked, doing nothing
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 shell am start -W -n com.ef.evc2015.kids/com.ef.evc2015.SplashActivity -S -a android.intent.action.MAIN -c android.intent.category.LAUNCHER -f 0x10200000'
[Appium] New AndroidDriver session created successfully, session 02d27be2-f024-4f9e-9c3e-2c919b262c6c added to master session list
[debug] [BaseDriver] Event 'newSessionStarted' logged at 1559563821389 (20:10:21 GMT+0800 (GMT+08:00))
[debug] [W3C (02d27be2)] Cached the protocol value 'W3C' for the new session 02d27be2-f024-4f9e-9c3e-2c919b262c6c
[debug] [W3C (02d27be2)] Responding to client with driver.createSession() result: {"capabilities":{"platform":"LINUX","webStorageEnabled":false,"takesScreenshot":true,"javascriptEnabled":true,"databaseEnabled":false,"networkConnectionEnabled":true,"locationContextEnabled":false,"warnings":{},"desired":{"platformName":"Android","deviceName":"f7cfcaf9","app":"C:\\1_Automation\\FM3\\app\\evckids-live-debug-1.2.0.3300.apk","automationName":"UiAutomator1","appPackage":"com.ef.evc2015.kids","appActivity":"com.ef.evc2015.SplashActivity","noReset":true,"chromeOptions.androidPackage":"com.android.chrome"},"platformName":"Android","deviceName":"f7cfcaf9","app":"C:\\1_Automation\\FM3\\app\\evckids-live-debug-1.2.0.3300.apk","automationName":"UiAutomator1","appPackage":"com.ef.evc2015.kids","appActivity":"com.ef.evc2015.SplashActivity","noReset":true,"chromeOptions.androidPackage":"com.android.chrome","deviceUDID":"f7cfcaf9","platformVersion":"7.1.1","deviceScreenSize":"1080x1920","deviceModel":"OPPO R11","deviceManufacturer":"OPPO"}}
[HTTP] <-- POST /wd/hub/session 200 5636 ms - 1055
[HTTP]
[HTTP] --> POST /wd/hub/session/02d27be2-f024-4f9e-9c3e-2c919b262c6c/element
[HTTP] {"using":"id","value":"com.ef.evc2015.kids:id/debugEntry"}
[debug] [W3C (02d27be2)] Calling AppiumDriver.findElement() with args: ["id","com.ef.evc2015.kids:id/debugEntry","02d27be2-f024-4f9e-9c3e-2c919b262c6c"]
[debug] [BaseDriver] Valid locator strategies for this request: xpath, id, class name, accessibility id, -android uiautomator
[debug] [BaseDriver] Waiting up to 0 ms for condition
[debug] [AndroidBootstrap] Sending command to android: {"cmd":"action","action":"find","params":{"strategy":"id","selector":"com.ef.evc2015.kids:id/debugEntry","context":"","multiple":false}}
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Got data from client: {"cmd":"action","action":"find","params":{"strategy":"id","selector":"com.ef.evc2015.kids:id/debugEntry","context":"","multiple":false}}
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Got command of type ACTION
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Got command action: find
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Finding 'com.ef.evc2015.kids:id/debugEntry' using 'ID' with the contextId: '' multiple: false
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Using: UiSelector[INSTANCE=0, RESOURCE_ID=com.ef.evc2015.kids:id/debugEntry]
[debug] [AndroidBootstrap] Received command result from bootstrap
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Returning result: {"status":0,"value":{"ELEMENT":"1"}}
[debug] [W3C (02d27be2)] Responding to client with driver.findElement() result: {"element-6066-11e4-a52e-4f735466cecf":"1","ELEMENT":"1"}
[HTTP] <-- POST /wd/hub/session/02d27be2-f024-4f9e-9c3e-2c919b262c6c/element 200 2339 ms - 67
[HTTP]
[HTTP] --> POST /wd/hub/session/02d27be2-f024-4f9e-9c3e-2c919b262c6c/element/1/click
[HTTP] {"id":"1"}
[debug] [W3C (02d27be2)] Calling AppiumDriver.click() with args: ["1","02d27be2-f024-4f9e-9c3e-2c919b262c6c"]
[debug] [AndroidBootstrap] Sending command to android: {"cmd":"action","action":"element:click","params":{"elementId":"1"}}
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Got data from client: {"cmd":"action","action":"element:click","params":{"elementId":"1"}}
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Got command of type ACTION
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Got command action: click
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Returning result: {"status":0,"value":true}
[debug] [AndroidBootstrap] Received command result from bootstrap
[debug] [W3C (02d27be2)] Responding to client with driver.click() result: true
[HTTP] <-- POST /wd/hub/session/02d27be2-f024-4f9e-9c3e-2c919b262c6c/element/1/click 200 319 ms - 14
[HTTP]
[HTTP] --> POST /wd/hub/session/02d27be2-f024-4f9e-9c3e-2c919b262c6c/touch/perform
[HTTP] {"actions":[{"action":"press","options":{"x":0,"y":500}},{"action":"wait","options":{"ms":0}},{"action":"moveTo","options":{"x":0,"y":0}},{"action":"release","options":{}}]}
[debug] [W3C (02d27be2)] Calling AppiumDriver.performTouch() with args: [[{"action":"press","options":{"x":0,"y":500}},{"action":"wait","options":{"ms":0}},{"action":"moveTo","options":{"x":0,"y":0}},{"action":"release","options":{}}],"02d27be2-f024-4f9e-9c3e-2c919b262c6c"]
[debug] [AndroidBootstrap] Sending command to android: {"cmd":"action","action":"swipe","params":{"startX":0,"startY":500,"endX":0,"endY":0,"steps":22}}
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Got data from client: {"cmd":"action","action":"swipe","params":{"startX":0,"startY":500,"endX":0,"endY":0,"steps":22}}
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Got command of type ACTION
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Got command action: swipe
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Display bounds: [0,0][1080,1920]
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Display bounds: [0,0][1080,1920]
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Swiping from [x=540.0, y=500.0] to [x=540.0, y=960.0] with steps: 22
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Returning result: {"status":0,"value":true}
[debug] [AndroidBootstrap] Received command result from bootstrap
[debug] [W3C (02d27be2)] Responding to client with driver.performTouch() result: true
[HTTP] <-- POST /wd/hub/session/02d27be2-f024-4f9e-9c3e-2c919b262c6c/touch/perform 200 366 ms - 14
[HTTP]
[HTTP] --> POST /wd/hub/session/02d27be2-f024-4f9e-9c3e-2c919b262c6c/element
[HTTP] {"using":"id","value":"com.ef.evc2015.kids:id/loadUrl"}
[debug] [W3C (02d27be2)] Calling AppiumDriver.findElement() with args: ["id","com.ef.evc2015.kids:id/loadUrl","02d27be2-f024-4f9e-9c3e-2c919b262c6c"]
[debug] [BaseDriver] Valid locator strategies for this request: xpath, id, class name, accessibility id, -android uiautomator
[debug] [BaseDriver] Waiting up to 0 ms for condition
[debug] [AndroidBootstrap] Sending command to android: {"cmd":"action","action":"find","params":{"strategy":"id","selector":"com.ef.evc2015.kids:id/loadUrl","context":"","multiple":false}}
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Got data from client: {"cmd":"action","action":"find","params":{"strategy":"id","selector":"com.ef.evc2015.kids:id/loadUrl","context":"","multiple":false}}
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Got command of type ACTION
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Got command action: find
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Finding 'com.ef.evc2015.kids:id/loadUrl' using 'ID' with the contextId: '' multiple: false
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Using: UiSelector[INSTANCE=0, RESOURCE_ID=com.ef.evc2015.kids:id/loadUrl]
[debug] [AndroidBootstrap] Received command result from bootstrap
[debug] [W3C (02d27be2)] Responding to client with driver.findElement() result: {"element-6066-11e4-a52e-4f735466cecf":"2","ELEMENT":"2"}
[HTTP] <-- POST /wd/hub/session/02d27be2-f024-4f9e-9c3e-2c919b262c6c/element 200 234 ms - 67
[HTTP]
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Returning result: {"status":0,"value":{"ELEMENT":"2"}}
[HTTP] --> POST /wd/hub/session/02d27be2-f024-4f9e-9c3e-2c919b262c6c/element/2/click
[HTTP] {"id":"2"}
[debug] [W3C (02d27be2)] Calling AppiumDriver.click() with args: ["2","02d27be2-f024-4f9e-9c3e-2c919b262c6c"]
[debug] [AndroidBootstrap] Sending command to android: {"cmd":"action","action":"element:click","params":{"elementId":"2"}}
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Got data from client: {"cmd":"action","action":"element:click","params":{"elementId":"2"}}
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Got command of type ACTION
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Got command action: click
[debug] [AndroidBootstrap] Received command result from bootstrap
[debug] [W3C (02d27be2)] Responding to client with driver.click() result: true
[HTTP] <-- POST /wd/hub/session/02d27be2-f024-4f9e-9c3e-2c919b262c6c/element/2/click 200 353 ms - 14
[HTTP]
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Returning result: {"status":0,"value":true}
[HTTP] --> GET /wd/hub/session/02d27be2-f024-4f9e-9c3e-2c919b262c6c/contexts
[HTTP] {}
[debug] [W3C (02d27be2)] Calling AppiumDriver.getContexts() with args: ["02d27be2-f024-4f9e-9c3e-2c919b262c6c"]
[debug] [AndroidDriver] Getting a list of available webviews
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 shell cat /proc/net/unix'
[debug] [AndroidDriver] WEBVIEW_27760 mapped to pid 27760
[debug] [AndroidDriver] Getting process name for webview
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 shell ps'
[debug] [AndroidDriver] Parsed pid: '27760' pkg: 'com.ef.evc2015.kids' from
[debug] [AndroidDriver]     USER      PID   PPID  VSIZE  RSS  FRZ          WCHAN              PC  NAME
[debug] [AndroidDriver]     u0_a315   27760 972   3165032 244528 efg           0 0000000000 S com.ef.evc2015.kids
[debug] [AndroidDriver] Returning process name: 'com.ef.evc2015.kids'
[debug] [AndroidDriver] Found webviews: ["WEBVIEW_com.ef.evc2015.kids"]
[debug] [AndroidDriver] Available contexts: ["NATIVE_APP","WEBVIEW_com.ef.evc2015.kids"]
[debug] [W3C (02d27be2)] Responding to client with driver.getContexts() result: ["NATIVE_APP","WEBVIEW_com.ef.evc2015.kids"]
[HTTP] <-- GET /wd/hub/session/02d27be2-f024-4f9e-9c3e-2c919b262c6c/contexts 200 225 ms - 54
[HTTP]
[HTTP] --> POST /wd/hub/session/02d27be2-f024-4f9e-9c3e-2c919b262c6c/context
[HTTP] {"name":"WEBVIEW_com.ef.evc2015.kids"}
[debug] [W3C (02d27be2)] Calling AppiumDriver.setContext() with args: ["WEBVIEW_com.ef.evc2015.kids","02d27be2-f024-4f9e-9c3e-2c919b262c6c"]
[debug] [AndroidDriver] Getting a list of available webviews
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 shell cat /proc/net/unix'
[debug] [AndroidDriver] WEBVIEW_27760 mapped to pid 27760
[debug] [AndroidDriver] Getting process name for webview
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 shell ps'
[debug] [AndroidDriver] Parsed pid: '27760' pkg: 'com.ef.evc2015.kids' from
[debug] [AndroidDriver]     USER      PID   PPID  VSIZE  RSS  FRZ          WCHAN              PC  NAME
[debug] [AndroidDriver]     u0_a315   27760 972   3147256 233560 efg           0 0000000000 S com.ef.evc2015.kids
[debug] [AndroidDriver] Returning process name: 'com.ef.evc2015.kids'
[debug] [AndroidDriver] Found webviews: ["WEBVIEW_com.ef.evc2015.kids"]
[debug] [AndroidDriver] Available contexts: ["NATIVE_APP","WEBVIEW_com.ef.evc2015.kids"]
[debug] [AndroidDriver] Connecting to chrome-backed webview context 'WEBVIEW_com.ef.evc2015.kids'
[debug] [AndroidDriver] A port was not given, using random free port: 8000
[debug] [AndroidDriver] Before starting chromedriver, androidPackage is com.ef.evc2015.kids
[debug] [Chromedriver] Changed state to 'starting'
[debug] [Chromedriver] Found 1 executable in 'C:\AppData\Roaming\npm\node_modules\appium\node_modules\appium-chromedriver\chromedriver\win'
[debug] [Chromedriver] The following Chromedriver executables were found:
[debug] [Chromedriver]     C:/Users/AppData/Roaming/npm/node_modules/appium/node_modules/appium-chromedriver/chromedriver/win/chromedriver.exe (minimum Chrome version '71.0.3578')
[debug] [ADB] Getting package info for 'com.android.chrome'
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 shell dumpsys package com.android.chrome'
[debug] [Chromedriver] Found Chrome bundle 'com.android.chrome' version '72.0.3626'
[debug] [Chromedriver] Found 1 Chromedriver executable capable of automating Chrome '72.0.3626'.
[debug] [Chromedriver] Choosing the most recent, 'C:/Users/AppData/Roaming/npm/node_modules/appium/node_modules/appium-chromedriver/chromedriver/win/chromedriver.exe'.
[debug] [Chromedriver] If a specific version is required, specify it with the `chromedriverExecutable`desired capability.
[Chromedriver] Set chromedriver binary as: C:/Users/AppData/Roaming/npm/node_modules/appium/node_modules/appium-chromedriver/chromedriver/win/chromedriver.exe
[debug] [Chromedriver] Killing any old chromedrivers, running: wmic process where "commandline like '%chromedriver.exe%--port=8000%'" delete
[debug] [Chromedriver] Successfully cleaned up old chromedrivers
[debug] [Chromedriver] Cleaning any old adb forwarded port socket connections
[debug] [ADB] List forwarding ports
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 forward --list'
[Chromedriver] Spawning chromedriver with: C:/Users/AppData/Roaming/npm/node_modules/appium/node_modules/appium-chromedriver/chromedriver/win/chromedriver.exe --url-base=wd/hub --port=8000 --adb-port=5037 --verbose
[debug] [Chromedriver] Chromedriver version: '2.46.628402'
[debug] [WD Proxy] Matched '/status' to command name 'getStatus'
[debug] [WD Proxy] Proxying [GET /status] to [GET http://127.0.0.1:8000/wd/hub/status] with no body
[debug] [WD Proxy] Got response with status 200: "{\"sessionId\":\"\",\"status\":0,\"value\":{\"build\":{\"version\":\"2.46.628402 (536cd7adbad73a3783fdc2cab92ab2ba7ec361e1)\"},\"message\":\"ChromeDriver ready for new sessions.\",\"os\":{\"arch\":\"x86_64\",\"name\":\"Windows NT\",\"version\":\"10.0.17763\"},\"ready\":true}}"
[debug] [WD Proxy] Determined that the downstream protocol for proxy is MJSONWP
[debug] [WD Proxy] Matched '/session' to command name 'createSession'
[debug] [WD Proxy] Proxying [POST /session] to [POST http://127.0.0.1:8000/wd/hub/session] with body: {"desiredCapabilities":{"chromeOptions":{"androidPackage":"com.ef.evc2015.kids","androidUseRunningApp":true,"androidDeviceSerial":"f7cfcaf9"},"loggingPrefs":{"browser":"ALL"}}}
[debug] [WD Proxy] Got response with status 200: {"sessionId":"5c7904000b838d6702cb74b0da954c25","status":13,"value":{"message":"unknown error: Failed to get PID for the following process: com.ef.evc2015.kids\n  (process name must be specified if not equal to package name)\n  (Driver info: chromedriver=2.46.628402 (536cd7adbad73a3783fdc2cab92ab2ba7ec361e1),platform=Windows NT 10.0.17763 x86_64)"}}
[WD Proxy] Got an unexpected response: {"sessionId":"5c7904000b838d6702cb74b0da954c25","status":13,"value":{"message":"unknown error: Failed to get PID for the following process: com.ef.evc2015.kids\n  (process name must be specified if not equal to package name)\n  (Driver info: chromedriver=2.46.628402 (536cd7adbad73a3783fdc2cab92ab...
[debug] [MJSONWP] Matched JSONWP error code 13 to UnknownError
[Chromedriver] Error: Failed to start Chromedriver session: An unknown server-side error occurred while processing the command. Original error: unknown error: Failed to get PID for the following process: com.ef.evc2015.kids
[Chromedriver]   (process name must be specified if not equal to package name)
[Chromedriver]   (Driver info: chromedriver=2.46.628402 (536cd7adbad73a3783fdc2cab92ab2ba7ec361e1),platform=Windows NT 10.0.17763 x86_64)
[Chromedriver]     at Object.wrappedLogger.errorAndThrow (C:\AppData\Roaming\npm\node_modules\appium\node_modules\appium-support\lib\logging.js:78:13)
[Chromedriver]     at errorAndThrow (C:\AppData\Roaming\npm\node_modules\appium\node_modules\appium-chromedriver\lib\chromedriver.js:493:13)
[debug] [WD Proxy] Matched '/session' to command name 'createSession'
[debug] [WD Proxy] Proxying [POST /session] to [POST http://127.0.0.1:8000/wd/hub/session] with body: {"desiredCapabilities":{"chromeOptions":{"androidPackage":"com.ef.evc2015.kids","androidUseRunningApp":true,"androidDeviceSerial":"f7cfcaf9"},"loggingPrefs":{"browser":"ALL"}}}
[debug] [WD Proxy] Got response with status 200: {"sessionId":"9189f89d79dc8b55a40f34e554ba85ad","status":13,"value":{"message":"unknown error: Failed to get PID for the following process: com.ef.evc2015.kids\n  (process name must be specified if not equal to package name)\n  (Driver info: chromedriver=2.46.628402 (536cd7adbad73a3783fdc2cab92ab2ba7ec361e1),platform=Windows NT 10.0.17763 x86_64)"}}
[WD Proxy] Got an unexpected response: {"sessionId":"9189f89d79dc8b55a40f34e554ba85ad","status":13,"value":{"message":"unknown error: Failed to get PID for the following process: com.ef.evc2015.kids\n  (process name must be specified if not equal to package name)\n  (Driver info: chromedriver=2.46.628402 (536cd7adbad73a3783fdc2cab92ab...
[debug] [MJSONWP] Matched JSONWP error code 13 to UnknownError
[Chromedriver] Error: Failed to start Chromedriver session: An unknown server-side error occurred while processing the command. Original error: unknown error: Failed to get PID for the following process: com.ef.evc2015.kids
[Chromedriver]   (process name must be specified if not equal to package name)
[Chromedriver]   (Driver info: chromedriver=2.46.628402 (536cd7adbad73a3783fdc2cab92ab2ba7ec361e1),platform=Windows NT 10.0.17763 x86_64)
[Chromedriver]     at Object.wrappedLogger.errorAndThrow (C:\AppData\Roaming\npm\node_modules\appium\node_modules\appium-support\lib\logging.js:78:13)
[Chromedriver]     at errorAndThrow (C:\AppData\Roaming\npm\node_modules\appium\node_modules\appium-chromedriver\lib\chromedriver.js:493:13)
[debug] [WD Proxy] Matched '/session' to command name 'createSession'
[debug] [WD Proxy] Proxying [POST /session] to [POST http://127.0.0.1:8000/wd/hub/session] with body: {"desiredCapabilities":{"chromeOptions":{"androidPackage":"com.ef.evc2015.kids","androidUseRunningApp":true,"androidDeviceSerial":"f7cfcaf9"},"loggingPrefs":{"browser":"ALL"}}}
[debug] [WD Proxy] Got response with status 200: {"sessionId":"93a69d2ef9eadac882959561a9fc0f40","status":13,"value":{"message":"unknown error: Failed to get PID for the following process: com.ef.evc2015.kids\n  (process name must be specified if not equal to package name)\n  (Driver info: chromedriver=2.46.628402 (536cd7adbad73a3783fdc2cab92ab2ba7ec361e1),platform=Windows NT 10.0.17763 x86_64)"}}
[WD Proxy] Got an unexpected response: {"sessionId":"93a69d2ef9eadac882959561a9fc0f40","status":13,"value":{"message":"unknown error: Failed to get PID for the following process: com.ef.evc2015.kids\n  (process name must be specified if not equal to package name)\n  (Driver info: chromedriver=2.46.628402 (536cd7adbad73a3783fdc2cab92ab...
[debug] [MJSONWP] Matched JSONWP error code 13 to UnknownError
[Chromedriver] Error: Failed to start Chromedriver session: An unknown server-side error occurred while processing the command. Original error: unknown error: Failed to get PID for the following process: com.ef.evc2015.kids
[Chromedriver]   (process name must be specified if not equal to package name)
[Chromedriver]   (Driver info: chromedriver=2.46.628402 (536cd7adbad73a3783fdc2cab92ab2ba7ec361e1),platform=Windows NT 10.0.17763 x86_64)
[Chromedriver]     at Object.wrappedLogger.errorAndThrow (C:\AppData\Roaming\npm\node_modules\appium\node_modules\appium-support\lib\logging.js:78:13)
[Chromedriver]     at errorAndThrow (C:\AppData\Roaming\npm\node_modules\appium\node_modules\appium-chromedriver\lib\chromedriver.js:493:13)
[debug] [WD Proxy] Matched '/session' to command name 'createSession'
[debug] [WD Proxy] Proxying [POST /session] to [POST http://127.0.0.1:8000/wd/hub/session] with body: {"desiredCapabilities":{"chromeOptions":{"androidPackage":"com.ef.evc2015.kids","androidUseRunningApp":true,"androidDeviceSerial":"f7cfcaf9"},"loggingPrefs":{"browser":"ALL"}}}
[debug] [WD Proxy] Got response with status 200: {"sessionId":"18756f548fe5c022417a4d7e854c4070","status":13,"value":{"message":"unknown error: Failed to get PID for the following process: com.ef.evc2015.kids\n  (process name must be specified if not equal to package name)\n  (Driver info: chromedriver=2.46.628402 (536cd7adbad73a3783fdc2cab92ab2ba7ec361e1),platform=Windows NT 10.0.17763 x86_64)"}}
[WD Proxy] Got an unexpected response: {"sessionId":"18756f548fe5c022417a4d7e854c4070","status":13,"value":{"message":"unknown error: Failed to get PID for the following process: com.ef.evc2015.kids\n  (process name must be specified if not equal to package name)\n  (Driver info: chromedriver=2.46.628402 (536cd7adbad73a3783fdc2cab92ab...
[debug] [MJSONWP] Matched JSONWP error code 13 to UnknownError
[Chromedriver] Error: Failed to start Chromedriver session: An unknown server-side error occurred while processing the command. Original error: unknown error: Failed to get PID for the following process: com.ef.evc2015.kids
[Chromedriver]   (process name must be specified if not equal to package name)
[Chromedriver]   (Driver info: chromedriver=2.46.628402 (536cd7adbad73a3783fdc2cab92ab2ba7ec361e1),platform=Windows NT 10.0.17763 x86_64)
[Chromedriver]     at Object.wrappedLogger.errorAndThrow (C:\AppData\Roaming\npm\node_modules\appium\node_modules\appium-support\lib\logging.js:78:13)
[Chromedriver]     at errorAndThrow (C:\AppData\Roaming\npm\node_modules\appium\node_modules\appium-chromedriver\lib\chromedriver.js:493:13)
[Chromedriver] Chromedriver exited unexpectedly with code null, signal SIGTERM
[debug] [Chromedriver] Changed state to 'stopped'
[Chromedriver] Error: Failed to start Chromedriver session: An unknown server-side error occurred while processing the command. Original error: unknown error: Failed to get PID for the following process: com.ef.evc2015.kids
[Chromedriver]   (process name must be specified if not equal to package name)
[Chromedriver]   (Driver info: chromedriver=2.46.628402 (536cd7adbad73a3783fdc2cab92ab2ba7ec361e1),platform=Windows NT 10.0.17763 x86_64)
[Chromedriver]     at Object.wrappedLogger.errorAndThrow (C:\AppData\Roaming\npm\node_modules\appium\node_modules\appium-support\lib\logging.js:78:13)
[Chromedriver]     at Chromedriver.errorAndThrow [as start] (C:\AppData\Roaming\npm\node_modules\appium\node_modules\appium-chromedriver\lib\chromedriver.js:441:11)
[debug] [W3C (02d27be2)] Encountered internal error running command: Error: Failed to start Chromedriver session: An unknown server-side error occurred while processing the command. Original error: unknown error: Failed to get PID for the following process: com.ef.evc2015.kids
[debug] [W3C (02d27be2)]   (process name must be specified if not equal to package name)
[debug] [W3C (02d27be2)]   (Driver info: chromedriver=2.46.628402 (536cd7adbad73a3783fdc2cab92ab2ba7ec361e1),platform=Windows NT 10.0.17763 x86_64)
[debug] [W3C (02d27be2)]     at Object.wrappedLogger.errorAndThrow (C:\AppData\Roaming\npm\node_modules\appium\node_modules\appium-support\lib\logging.js:78:13)
[debug] [W3C (02d27be2)]     at Chromedriver.errorAndThrow [as start] (C:\AppData\Roaming\npm\node_modules\appium\node_modules\appium-chromedriver\lib\chromedriver.js:441:11)
[HTTP] <-- POST /wd/hub/session/02d27be2-f024-4f9e-9c3e-2c919b262c6c/context 500 4075 ms - 1351
[HTTP]
[HTTP] --> GET /wd/hub/session/02d27be2-f024-4f9e-9c3e-2c919b262c6c/alert/text
[HTTP] {}
[debug] [W3C (02d27be2)] Calling AppiumDriver.getAlertText() with args: ["02d27be2-f024-4f9e-9c3e-2c919b262c6c"]
[debug] [W3C (02d27be2)] Encountered internal error running command: NotYetImplementedError: Method has not yet been implemented
[debug] [W3C (02d27be2)]     at AndroidDriver.getAlertText (C:\AppData\Roaming\npm\node_modules\appium\node_modules\appium-android-driver\lib\commands\alert.js:7:9)
[debug] [W3C (02d27be2)]     at curCommandCancellable._bluebird.default.resolve.then (C:\AppData\Roaming\npm\node_modules\appium\node_modules\appium-base-driver\lib\basedriver\driver.js:301:66)
[debug] [W3C (02d27be2)]     at tryCatcher (C:\AppData\Roaming\npm\node_modules\appium\node_modules\bluebird\js\release\util.js:16:23)
[debug] [W3C (02d27be2)]     at Promise._settlePromiseFromHandler (C:\AppData\Roaming\npm\node_modules\appium\node_modules\bluebird\js\release\promise.js:512:31)
[debug] [W3C (02d27be2)]     at Promise._settlePromise (C:\AppData\Roaming\npm\node_modules\appium\node_modules\bluebird\js\release\promise.js:569:18)
[debug] [W3C (02d27be2)]     at Promise._settlePromiseCtx (C:\AppData\Roaming\npm\node_modules\appium\node_modules\bluebird\js\release\promise.js:606:10)
[debug] [W3C (02d27be2)]     at _drainQueueStep (C:\AppData\Roaming\npm\node_modules\appium\node_modules\bluebird\js\release\async.js:142:12)
[debug] [W3C (02d27be2)]     at _drainQueue (C:\AppData\Roaming\npm\node_modules\appium\node_modules\bluebird\js\release\async.js:131:9)
[debug] [W3C (02d27be2)]     at Async._drainQueues (C:\AppData\Roaming\npm\node_modules\appium\node_modules\bluebird\js\release\async.js:147:5)
[debug] [W3C (02d27be2)]     at Immediate.Async.drainQueues (C:\AppData\Roaming\npm\node_modules\appium\node_modules\bluebird\js\release\async.js:17:14)
[debug] [W3C (02d27be2)]     at runCallback (timers.js:705:18)
[debug] [W3C (02d27be2)]     at tryOnImmediate (timers.js:676:5)
[debug] [W3C (02d27be2)]     at processImmediate (timers.js:658:5)
[HTTP] <-- GET /wd/hub/session/02d27be2-f024-4f9e-9c3e-2c919b262c6c/alert/text 404 7 ms - 1869
[HTTP]
[HTTP] --> GET /wd/hub/session/02d27be2-f024-4f9e-9c3e-2c919b262c6c/alert/text
[HTTP] {}
[debug] [W3C (02d27be2)] Calling AppiumDriver.getAlertText() with args: ["02d27be2-f024-4f9e-9c3e-2c919b262c6c"]
[debug] [W3C (02d27be2)] Encountered internal error running command: NotYetImplementedError: Method has not yet been implemented
[debug] [W3C (02d27be2)]     at AndroidDriver.getAlertText (C:\AppData\Roaming\npm\node_modules\appium\node_modules\appium-android-driver\lib\commands\alert.js:7:9)
[debug] [W3C (02d27be2)]     at curCommandCancellable._bluebird.default.resolve.then (C:\AppData\Roaming\npm\node_modules\appium\node_modules\appium-base-driver\lib\basedriver\driver.js:301:66)
[debug] [W3C (02d27be2)]     at tryCatcher (C:\AppData\Roaming\npm\node_modules\appium\node_modules\bluebird\js\release\util.js:16:23)
[debug] [W3C (02d27be2)]     at Promise._settlePromiseFromHandler (C:\AppData\Roaming\npm\node_modules\appium\node_modules\bluebird\js\release\promise.js:512:31)
[debug] [W3C (02d27be2)]     at Promise._settlePromise (C:\AppData\Roaming\npm\node_modules\appium\node_modules\bluebird\js\release\promise.js:569:18)
[debug] [W3C (02d27be2)]     at Promise._settlePromiseCtx (C:\AppData\Roaming\npm\node_modules\appium\node_modules\bluebird\js\release\promise.js:606:10)
[debug] [W3C (02d27be2)]     at _drainQueueStep (C:\AppData\Roaming\npm\node_modules\appium\node_modules\bluebird\js\release\async.js:142:12)
[debug] [W3C (02d27be2)]     at _drainQueue (C:\AppData\Roaming\npm\node_modules\appium\node_modules\bluebird\js\release\async.js:131:9)
[debug] [W3C (02d27be2)]     at Async._drainQueues (C:\AppData\Roaming\npm\node_modules\appium\node_modules\bluebird\js\release\async.js:147:5)
[debug] [W3C (02d27be2)]     at Immediate.Async.drainQueues (C:\AppData\Roaming\npm\node_modules\appium\node_modules\bluebird\js\release\async.js:17:14)
[debug] [W3C (02d27be2)]     at runCallback (timers.js:705:18)
[debug] [W3C (02d27be2)]     at tryOnImmediate (timers.js:676:5)
[debug] [W3C (02d27be2)]     at processImmediate (timers.js:658:5)
[HTTP] <-- GET /wd/hub/session/02d27be2-f024-4f9e-9c3e-2c919b262c6c/alert/text 404 3 ms - 1869
[HTTP]
[HTTP] --> GET /wd/hub/session/02d27be2-f024-4f9e-9c3e-2c919b262c6c/url
[HTTP] {}
[debug] [W3C (02d27be2)] Calling AppiumDriver.getUrl() with args: ["02d27be2-f024-4f9e-9c3e-2c919b262c6c"]
[debug] [W3C (02d27be2)] Encountered internal error running command: NotYetImplementedError: Method has not yet been implemented
[debug] [W3C (02d27be2)]     at AndroidDriver.executeCommand (C:\AppData\Roaming\npm\node_modules\appium\node_modules\appium-base-driver\lib\basedriver\driver.js:266:13)
[debug] [W3C (02d27be2)]     at AppiumDriver.executeCommand (C:\AppData\Roaming\npm\node_modules\appium\lib\appium.js:485:36)
[debug] [W3C (02d27be2)]     at process._tickCallback (internal/process/next_tick.js:68:7)
[HTTP] <-- GET /wd/hub/session/02d27be2-f024-4f9e-9c3e-2c919b262c6c/url 404 1 ms - 529
[HTTP]
[HTTP] --> GET /wd/hub/session/02d27be2-f024-4f9e-9c3e-2c919b262c6c/title
[HTTP] {}
[debug] [W3C (02d27be2)] Calling AppiumDriver.title() with args: ["02d27be2-f024-4f9e-9c3e-2c919b262c6c"]
[debug] [W3C (02d27be2)] Encountered internal error running command: NotYetImplementedError: Method has not yet been implemented
[debug] [W3C (02d27be2)]     at AndroidDriver.executeCommand (C:\AppData\Roaming\npm\node_modules\appium\node_modules\appium-base-driver\lib\basedriver\driver.js:266:13)
[debug] [W3C (02d27be2)]     at AppiumDriver.executeCommand (C:\AppData\Roaming\npm\node_modules\appium\lib\appium.js:485:36)
[debug] [W3C (02d27be2)]     at process._tickCallback (internal/process/next_tick.js:68:7)
[HTTP] <-- GET /wd/hub/session/02d27be2-f024-4f9e-9c3e-2c919b262c6c/title 404 1 ms - 529
[HTTP]
[HTTP] --> GET /wd/hub/session/02d27be2-f024-4f9e-9c3e-2c919b262c6c/screenshot
[HTTP] {}
[debug] [W3C (02d27be2)] Calling AppiumDriver.getScreenshot() with args: ["02d27be2-f024-4f9e-9c3e-2c919b262c6c"]
[debug] [W3C (02d27be2)] Responding to client with driver.getScreenshot() result: "iVBORw0KGgoAAAANSUhEUgAAB4AAAAQ4CAYAAADo08FDAAIIhUlEQVR4AezBCZyVBcHw7f995pzZWIZ9GfZFFhWIJDfUTEOixDTFLcNcWuxJS98ot6fQJH3TSivrjZAo00dR8UHNNNx3R1RARdlkZ1hmYAZn3+7vx/E7IwgzzAwzAxz/1xWED/YLkSRJkiRJkiRJkiQd9CJIkiRJkiRJkiRJkpJCBEmSJEmSJEmSJElSUoggSZIkSZIkSZIkSUoKESRJkiRJkiRJkiRJSSGCJEmSJEmSJEmSJCkpRJAkSZIkSZIkSZIkJYUIkiRJkiRJkiRJkqSkEEGSJEmSJEmSJEmSlBQiSJIkSZIkSZIkSZKSQgRJkiRJkiRJkiRJUlKIIEmSJElKTpEYZPSk1aRkoHrE2kNqB4i1RftZagdI6wJpnZEkSZKkZBNFkiRJkpoi1h56nAhdj4Etr8DaR2l1w38EPb4ERSvhjauAkM+caBuoKmaP2g2CI26FqhLIexUW385uuh0P1SWQ/yb7pO0AOOIW+OhDWDsXtrxGnXp8CQ65lLhF06BwMZ8JR/0eUjtD4fvw5s+AkM+kk+YCEVj3KCydzn5x2BToOBLK8+Dli4GQFtHteOj8efjgTgiraLAgCp//FZRtgU3PQ/5bEFbRIBk9oMuRsPYRJEmSJH02RZEkSZKkhsrsDZ0+B51GQ8cRkJJBXPtDoHgtbF0An5sKnY6gSTY+C4t/S4MFKRCkQJBCXJs+0G4QzWLbu1CexwFv9E2Q3gU+WgULb4Cwilpt+hAXzYSaanYz5PvQ+6tQuR0WTIWPltNkfSZCSiZ0OBxW3U+9IlGItScuSEFKStmnwLDLiavYCivupsF6fAmyhkPWcMjMhvy3aJBOo+CwKRDLgrQusHwmkiRJkj57okiSJElSfXqOg65HQdsBkN6N3ZRthvw3oaqY/a7zGBh8Mc1i0S+hPI9dfPEBSEmnVSy4HrYupF6Z2dBuIBCBSBTCanaR0ZNaxavZzdY3IfsU...
[HTTP] <-- GET /wd/hub/session/02d27be2-f024-4f9e-9c3e-2c919b262c6c/screenshot 200 848 ms - 177760
[HTTP]
[HTTP] --> DELETE /wd/hub/session/02d27be2-f024-4f9e-9c3e-2c919b262c6c
[HTTP] {}
[debug] [W3C (02d27be2)] Calling AppiumDriver.deleteSession() with args: ["02d27be2-f024-4f9e-9c3e-2c919b262c6c"]
[debug] [BaseDriver] Event 'quitSessionRequested' logged at 1559563850833 (20:10:50 GMT+0800 (GMT+08:00))
[Appium] Removing session 02d27be2-f024-4f9e-9c3e-2c919b262c6c from our master session list
[debug] [AndroidDriver] Shutting down Android driver
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 shell am force-stop com.ef.evc2015.kids'
[debug] [ADB] Pressing the HOME button
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 shell input keyevent 3'
[debug] [AndroidBootstrap] Sending command to android: {"cmd":"shutdown"}
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Got data from client: {"cmd":"shutdown"}
[debug] [AndroidBootstrap] Received command result from bootstrap
[debug] [UiAutomator] Shutting down UiAutomator
[debug] [UiAutomator] Moving to state 'stopping'
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Got command of type SHUTDOWN
[debug] [AndroidBootstrap] [BOOTSTRAP LOG] [debug] Returning result: {"status":0,"value":"OK, shutting down"}
[debug] [UiAutomator] UiAutomator shut down normally
[debug] [UiAutomator] Moving to state 'stopped'
[debug] [ADB] Attempting to kill all uiautomator processes
[debug] [ADB] Getting IDs of all 'uiautomator' processes
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 shell pgrep \^uiautomator\$'
[ADB] No 'uiautomator' process has been found
[debug] [UiAutomator] Moving to state 'stopped'
[debug] [Logcat] Stopping logcat capture
[debug] [ADB] Running 'C:\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s f7cfcaf9 shell am force-stop io.appium.unlock'
[debug] [AndroidDriver] Not cleaning generated files. Add `clearSystemFiles` capability if wanted.
[debug] [BaseDriver] Event 'quitSessionFinished' logged at 1559563853463 (20:10:53 GMT+0800 (GMT+08:00))
[debug] [W3C (02d27be2)] Received response: null
[debug] [W3C (02d27be2)] But deleting session, so not returning
[debug] [W3C (02d27be2)] Responding to client with driver.deleteSession() result: null
[HTTP] <-- DELETE /wd/hub/session/02d27be2-f024-4f9e-9c3e-2c919b262c6c 200


And I failed to get any info of my OPPO R11 by visiting: chrome://inspect/#devices

**Comment by mykola-mokhnach** [comment]
> And I failed to get any info of my OPPO R11 by visiting: chrome://inspect/#devices

If it is not possible to get the web views in Chrome then chrome driver won't be able to get them at all. Please make sure you have all the mandatory config options set to access the stuff from Chrome debugger first

**Comment by cindysongzi** [comment]
I can get chrome version by visiting chrome://inspect/#devices. I replaced chromedriver manually and restated my machine. It works on my Samsung Tablet.... Close the ticket.
But it stills says An unknown server-side error occurred while processing the command. Original error: Failed to start Chromedriver session: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to remote server. Original error: Error: connect ECONNREFUSED 127.0.0.1:8000 on my OPPO phone. I'm going to raise another issue.

**Comment by JohnnyVip** [comment]
raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Failed to start
Chromedriver session: An unknown server-side error occurred while processing the command. Original error: unknown error: unrecognized browser version: 40.22
14.121
What should I do ？
